src/goto-programs/remove_ternary.h

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************\
+
+Module: Remove function returns
+
+Author: Daniel Kroening
+
+Date:   September 2009
+
+\*******************************************************************/
+
+#ifndef CPROVER_REMOVE_TERNARY_H
+#define CPROVER_REMOVE_TERNARY_H
+
+#include <goto-programs/goto_model.h>
+
+class remove_ternaryt
+{
```

Comment:
This class definition should probably be moved to the .cpp file as is done in other transformations.